assets/js/base/utils/address.js

```diff
@@ -10,5 +10,5 @@ export const pluckAddress = ( { country, state, city, postcode } ) => ( {
 	country,
 	state,
 	city,
-	postcode,
+	postcode: postcode.replace( ' ', '' ),
```

Comment:
Since this function is used to pluck address data for comparison (which in turn is used to determine whether to do a http request to update shipping rates), normalization is needed here to cover cases where postal code might have spaces in it. This was a not so fun bug I had to troubleshoot while developing this pull.

Reply:
Qu: does this also need converting to upper case?

Reply:
Ya probably should, good catch! (fixed in 17de9868e9)